File: Lista de exercicio/projetoPYTHON/questao8/questao8.py
class Empregado:
    def __init__(self, nome, cargo, salario):
        self.nome = nome
        self.cargo = cargo
        self.salario = salario

class Empresa:
    def __init__(self):
        self.empregados = []

    def adicionar_empregado(self, empregado):
        self.empregados.append(empregado)

def executar_questao8():
    empregado1 = Empregado("Matheus", "Desenvolvedor", 3000)
    empregado2 = Empregado("Lauane", "RH", 3500)
    empregado3 = Empregado("Otavio", "Analista de Dados", 4000)

    empresa = Empresa()

    empresa.adicionar_empregado(empregado1)
    empresa.adicionar_empregado(empregado2)
    empresa.adicionar_empregado(empregado3)

    for emp in empresa.empregados:
        print(f"Empregado: {emp.nome}, Cargo: {emp.cargo}, Salario: R${emp.salario}")

File: Lista de exercicio/projetoJAVA/src/projetojava/questao8.java
package projetojava;

import java.util.ArrayList;
import java.util.List;

class Empregado {
    String nome;
    String cargo;
    double salario;

    Empregado(String nome, String cargo, double salario) {
        this.nome = nome;
        this.cargo = cargo;
        this.salario = salario;
    }
}

class Empresa {
    List<Empregado> empregados;

    Empresa() {
        empregados = new ArrayList<>();
    }

    void adicionarEmpregado(Empregado empregado) {
        empregados.add(empregado);
    }
}

public class questao8 {
    public static void main(String[] args) {
        Empregado empregado1 = new Empregado("Matheus", "Desenvolvedor", 3000);
        Empregado empregado2 = new Empregado("Lauane", "RH", 3500);
        Empregado empregado3 = new Empregado("Otavio", "Analista de Dados", 4000);

        Empresa empresa = new Empresa();

        empresa.adicionarEmpregado(empregado1);
        empresa.adicionarEmpregado(empregado2);
        empresa.adicionarEmpregado(empregado3);

        for (Empregado emp : empresa.empregados) {
            System.out.println("Empregado: " + emp.nome + ", Cargo: " + emp.cargo + ", Salario: R$" + emp.salario);
        }
    }
}
